libs/clients/p-card/src/lib/pCardClient.config.ts

```diff
@@ -12,7 +12,7 @@ export const PCardClientConfig = defineConfig<z.infer<typeof schema>>({
   schema,
   load: (env) => ({
     xRoadServicePath: env.required(
-      'XROAD_DISTRICT_COMMISSIONERS_PATH',
+      'XROAD_DISTRICT_COMMISSIONERS_PCARD_PATH',
```

Comment:
Make sure that this env variable exists in all environments.

Reply:
It's statically declared in the `/dsl/xroad.ts` and its correct there. However, the `DistrictCommissionersLicenses` paths were wrong so i fixed that instead